runtime/compiler/control/J9Options.cpp

```diff
@@ -1328,26 +1328,6 @@ J9::Options::fePreProcess(void * base)
          }
       }
 
-
-   // The -Xlp option may have a numeric argument but we don't care what
-   // it is because it applies to large data pages.
-   //
-   if (FIND_ARG_IN_VMARGS(EXACT_MATCH, "-Xlp", 0) >= 0)
-      {
-      self()->setOption(TR_EnableLargePages);
-      self()->setOption(TR_EnableLargeCodePages);
-      }
```

Comment:
@gita-omr This was totally removed, since there doesn't exist any logic for `TR_EnableLargePages` `TR_EnableLargeCodePages`.